### PR TITLE
Make the recordings symlink be generated by utility not DataPlayback

### PR DESCRIPTION
**Pull request by TrentHouliston**
Currently if you make a role that tries to do something with the recordings directory in docker, it will only find it if you happend to build the DataPlayback module in a role.

This ensures it's always created by making it a dependency of utility rather than that module

### CODE REVIEW

File: shared/utility/libraries.cmake [hunk 1]
```diff
@@ -21,4 +21,13 @@ if(CMAKE_BUILD_TYPE STREQUAL "Debug")
   target_link_libraries(nuclear_utility PUBLIC libbacktrace::libbacktrace ${CMAKE_DL_LIBS})
 endif()
 
+# Generate the decoders for each of the message types we know
+add_custom_command(
+  OUTPUT "${source_file}"
```
Comment (KipHamiltons) [inline shared/utility/libraries.cmake:26]: I am probably missing something, but `${source_file}` is undefined here, so I don't know what `add_custom_command` is meant to output

File: shared/utility/libraries.cmake [hunk 2]
```diff
@@ -21,4 +21,13 @@ if(CMAKE_BUILD_TYPE STREQUAL "Debug")
...
+  COMMAND ${PYTHON_EXECUTABLE} ARGS "${CMAKE_CURRENT_SOURCE_DIR}/src/generate_dataplayback.py"
+          "${PROJECT_BINARY_DIR}/nuclear/message/python" "${source_file}"
```
Comment (KipHamiltons) [inline shared/utility/libraries.cmake:28]: that also makes this line strange

**Review by KipHamiltons — COMMENTED** [review]
Bit confused about this one

File: shared/utility/libraries.cmake (hunk 1 above)
Comment (TrentHouliston) [reply shared/utility/libraries.cmake:26]: I clearly copied the wrong add_custom_command.... this is what happens when you do too many PRs at the same time

**Review by KipHamiltons — APPROVED** [review]
Presuming this works, LGTM

**Pull request merged**